var/spack/repos/builtin/packages/shamrock/package.py

```diff
@@ -0,0 +1,115 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Shamrock(CMakePackage):
+    """The Shamrock exascale framework for astrophysics"""
+
+    homepage = "https://shamrock-code.github.io/"
+    url = "https://github.com/Shamrock-code/Shamrock/releases/download/v2025.05.0/shamrock-2025.05.0.tar"
+    git = "https://github.com/Shamrock-code/Shamrock.git"
+
+    maintainers("tdavidcl")
+
+    license("CeCILL Free Software License Agreement v2.1", checked_by="tdavidcl")
+
+    version("main", branch="main", submodules=True)
+
+    version("2025.05.0", sha256="59d5652467fd9453a65ae7b48e0c9b7d4162edc8df92e09d08dcc5275407a897")
+
+    depends_on("c", type="build")
+    depends_on("cxx", type="build")
+
+    variant("shared", default=True, description="Enables the build of shared libraries")
+    variant("testing", default=True, description="Enables the build of shared libraries")
+    variant("pybindings", default=True, description="Install python bindings")
+
+    depends_on("sycl")
+    depends_on("mpi")
+    depends_on("python")
+    extends("python", when="+pybindings")
+
+    depends_on("ninja", type="build", when="generator=ninja")
```

Comment:
This is also in the base class, so I think we can delete it:
```suggestion
```

Reply:
Without this the build default to make which is ~2 times slower since we have many libraries. Would that be ok to keep ninja like so ?

Reply:
How does this influence the default? What you want is likely:
```suggestion
    generator("ninja")
```
like other packages do.

Reply:
I don't remember where i got this line from. I will change it to `generator("ninja")`